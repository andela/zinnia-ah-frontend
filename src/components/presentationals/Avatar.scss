.avatar{
	width:45px;
	height:45px;
	border-radius:100%;
}
.dropdown {
  height: 100%;
}

