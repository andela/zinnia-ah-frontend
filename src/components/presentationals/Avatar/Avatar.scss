.avatar {
  border-radius: 100%;
  height: 2.8125rem;
  width: 2.8125rem;
}

.avatar-xl {
  border-radius: 100%;
  height: 12.5rem;
  width: 12.5rem;
}

.dropdown {
  height: 100%;
}
