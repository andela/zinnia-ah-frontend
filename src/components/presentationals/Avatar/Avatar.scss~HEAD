.avatar {
  border-radius: 100%;
  height: 2.8125rem;
  width: 2.8125rem;
}

.dropdown {
  height: 100%;
}
