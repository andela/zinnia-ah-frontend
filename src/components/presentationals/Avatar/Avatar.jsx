import React from 'react';
import PropTypes from 'prop-types';

import './Avatar.scss';

const Avatar = ({ avatarUrl }) => {
  return <img src={avatarUrl} alt="avatar" className="avatar" />;
};

Avatar.propTypes = {
  avatarUrl: PropTypes.string,
};

export default Avatar;
