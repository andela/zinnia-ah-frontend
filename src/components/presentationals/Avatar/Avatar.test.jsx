import React from 'react';
import { mount } from 'enzyme';

import Avatar from './Avatar';

describe('Avatar', () => {
  it('should render correctly', () => {
    const component = mount(
      <Avatar className="btn" src="http://dummyimage.jpg" />,
    );
    expect(component).toMatchSnapshot();
  });
});
