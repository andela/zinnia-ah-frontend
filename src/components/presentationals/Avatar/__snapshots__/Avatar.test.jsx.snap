// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render correctly 1`] = `
<Avatar
  className="btn"
  src="http://dummyimage.jpg"
>
  <img
    alt="avatar"
    className="avatar btn"
  />
</Avatar>
`;
